src/liboslexec/splineimpl.h

```diff
@@ -43,45 +47,100 @@ namespace Spline {
 
 struct SplineBasis {
    int      basis_step;
-   Matrix44 basis;
+   float    basis[4][4];
```

Comment:
Regretfully, I was a little lazy when adapting ``ImathMatrix.h`` and didn't add the ``HOST_DEVICE`` decorators to the ``Matrix44`` templates. But it should be possible to use ``Matrix44`` for CUDA just like the lower-dimensional variants. It's probably not worth a PR in itself, but I can forward you a patch with a fully-dressed header.

Reply:
What we need to do is get a PR into OpenEXR itself that adds the decorators to all their matrix and vector classes at the source so that they are safe to compile for Cuda.